src/ctia/http/routes/common.clj

```diff
@@ -117,19 +117,20 @@
      (cond-> {}
        (seq date-range) (assoc-in [:range date-field] date-range)
        (seq filter-map) (assoc :filter-map filter-map)
```

Comment:
few lines above filter-map is prepared as below:
``` clojure
filter-map (apply dissoc search-params filter-map-search-options)
```
with `filter-map-search-options` (probably not a good name) containing the search control parameters:
``` clojure
(def search-options [:sort_by
                     :sort_order
                     :offset
                     :limit
                     :fields
                     :search_after])
                     
(def filter-map-search-options
  (conj search-options :query :from :to))
```

We need to add :query_mode to `search-options`